ui/app/pages/send/send.utils.js

```diff
@@ -72,7 +71,7 @@ function isBalanceSufficient ({
     {
       value: totalAmount,
       fromNumericBase: 'hex',
-      conversionRate: Number(amountConversionRate) || conversionRate,
```

Comment:
This seems to be the only place `amountConversionRate` is used. It falls back to `conversionRate` though, which is also what the `getAmountConversionRate` selector falls back to if no token is selected.

So this all seems safe to delete - it should be functionally equivalent.